R/plot-population-time-profile.R

```diff
@@ -57,13 +57,16 @@ plotPopulationTimeProfile <- function(dataCombined,
   individualTimeProfilePlotConfiguration$export <- defaultInternalPlotConfiguration$export
 
   # If axes labels haven't been specified, create them using dimensions and units.
+  xUnitString <- ifelse(unique(df$xUnit) == "", unique(df$xUnit), paste0(" [", unique(df$xUnit), "]"))
+  yUnitString <- ifelse(unique(df$yUnit) == "", unique(df$yUnit), paste0(" [", unique(df$yUnit), "]"))
```

Comment:
Sorry if obvious. Do you know at that stage that you have unique unit and dimension? is this the way the df is constructed?

Reply:
All of this is happening after `.unitCoverter()` has modified the data frame `df` to have the same `xUnit` and `yUnit` across datasets.

Reply:
ok , So what about you specify in the comment sthg along those lines? e.g. //unitCOnvert has ensured that we have only one single unit for x and y so we can safely take the unqiue/first etc..

Reply:
Good idea. Added a comment now.